server/service/src/requisition/request_requisition/insert.rs

```diff
@@ -58,6 +61,19 @@ pub fn insert_request_requisition(
                 .ok_or(OutError::NewlyCreatedRequisitionDoesNotExist)
         })
         .map_err(|error| error.to_inner_error())?;
+
+    log_entry(
+        &ctx.connection,
+        &LogRow {
+            id: uuid(),
+            r#type: LogType::RequisitionCreated,
+            user_id: Some(user_id.to_string()),
+            store_id: Some(store_id.to_string()),
+            record_id: Some(requisition.requisition_row.id.to_string()),
+            datetime: requisition.requisition_row.created_datetime,
```

Comment:
Sorry, I haven't really reviewed these PRs, one improvement (once store_id and user_id are context), is to just pass context, record_id and r#type to log_entry